koji/collection-mash-split.py

```diff
@@ -400,12 +400,15 @@ def main():
 
         CONFIG["comps_path"] = "/mnt/koji/mash/comps/foreman"
 
-        mashes = [MashConfig(collection, version, "rhel7-dist", "rhel7", "RHEL/7")]
         if collection == 'foreman' and version not in ('3.1', '3.0', '2.5'):
             modulemd_suffix = 'el8'
         else:
             modulemd_suffix = None
-        mashes.append(MashConfig(collection, version, "el8", "el8", "RHEL/8", modulemd_suffix=modulemd_suffix, modulemd_version=modulemd_version, modulemd_context=modulemd_context))
+
+        mashes = [MashConfig(collection, version, "el8", "el8", "RHEL/8", modulemd_suffix=modulemd_suffix, modulemd_version=modulemd_version, modulemd_context=modulemd_context)]
```

Comment:
It's my (longer term) desire to align things and I'd like to change `RHEL/8` to `el8`. I wonder if now is a good moment. We'd have to change forklift to stop using the older repos, but on the other hand it'd ensure you're not using stale EL7 content.